great_expectations/datasource/fluent/pandas_datasource.pyi

```diff
@@ -142,25 +143,35 @@ class PandasDatasource(_PandasDatasource):
     type: Literal["pandas"]
     assets: Dict[str, _PandasDataAsset]
     def test_connection(self, test_assets: bool = ...) -> None: ...
-    def _get_validator(self, asset: _PandasDataAsset) -> Validator: ...
+    def _get_validator(self, asset: _PandasDataAssetT) -> Validator: ...
     def add_dataframe_asset(
-        self, name: str, dataframe: pd.DataFrame
+        self,
+        name: str,
+        *,
+        dataframe: pd.DataFrame,
+        batch_metadata: Optional[BatchMetadata] = ...,
     ) -> DataFrameAsset: ...
     def read_dataframe(
-        self, dataframe: pd.DataFrame, asset_name: Optional[str] = ...
+        self,
+        dataframe: pd.DataFrame,
+        *,
+        asset_name: Optional[str] = ...,
+        batch_metadata: Optional[BatchMetadata] = ...,
     ) -> Validator: ...
     def add_clipboard_asset(
         self,
         name: str,
-        order_by: typing.List[Sorter] = ...,
```

Comment:
`order_by` does not apply to `PandasDatasource`, as it is a single-batch `Datasource`.